repo/packages/N/netsil/0/config.json

```diff
@@ -62,7 +62,7 @@
           "type": "integer"
```

Comment:
Looks like there are a lot of changes over the previous package. Please create another package version 
repo/packages/N/netsil/1